tests/NewCypress/cypress/e2e/regression/search-results-page.cy.ts

```diff
@@ -34,7 +34,7 @@ describe("Testing the components of the search results page", () => {
             .checkSearchResultsReturned("East")
     });
 
-    it.only("Validates that it reutrns the amount of results stated in the search text", () => {
+    it("Validates that it reutrns the amount of results stated in the search text", () => {
```

Comment:
Missed this one before, small typo
```suggestion
    it("Validates that it returns the amount of results stated in the search text", () => {
```